starter-core/src/main/java/io/micronaut/starter/feature/dependencies/DependenciesFeature.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.dependencies;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.feature.Features;
+import io.micronaut.starter.feature.test.TestFeature;
+import io.micronaut.starter.options.BuildTool;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public interface DependenciesFeature {
```

Comment:
Why adding these methods here? Could they not belong to `Features`?

Reply:
This no longer applies